app/controllers/admin/export/document_controller.rb

```diff
@@ -24,6 +27,24 @@ def edition_associations(edition)
     associations.each do |association|
       output[:associations][association] = edition.public_send(association)
     end
+    output[:whitehall_admin_links] = resolve_whitehall_admin_links(edition.body)
     output
   end
+
+  def resolve_whitehall_admin_links(body)
+    whitehall_links = whitehall_admin_links(body)
+    public_urls = []
+    whitehall_links.each do |link|
+      public_url = public_url_for_admin_link(link)
+      public_urls.push("whitehall_admin_url": link, "public_url": public_url)
+    end
+    public_urls
+  end
+
+  def public_url_for_admin_link(url)
+    edition = Whitehall::AdminLinkLookup.find_edition(url)
+    if edition.present? && edition.linkable?
```

Comment:
What does linkable mean in this context? I ask as I'm wondering if we want to link even if it's not linkable so we have something to fill in and may be a 404 or a redirect but that seems to reveal more than if we have a nil response

Reply:
It means the document has been marked as published.  In this case, we probably don't want to have this check, so we still get the forecast public URL for documents that are still in draft.